p2panda-spaces/src/manager.rs

```diff
@@ -251,6 +233,59 @@ where
         Ok(events)
     }
 
+    // @TODO: Make it work without async
+    /// The public key of the local actor.
+    pub async fn id(&self) -> ActorId {
+        let manager = self.inner.read().await;
+        manager.identity.id()
+    }
+
+    /// The local actor id and their long-term key bundle.
+    pub async fn me(&self) -> Result<Member, ManagerError<ID, S, K, M, C, RS>> {
+        let mut manager = self.inner.write().await;
+        manager
+            .identity
+            .me()
+            .await
+            .map_err(ManagerError::IdentityManager)
+    }
+
+    /// Register a member with long-term key bundle material.
+    pub async fn register_member(
+        &self,
+        member: &Member,
+    ) -> Result<(), ManagerError<ID, S, K, M, C, RS>> {
+        let mut manager = self.inner.write().await;
+        manager
+            .identity
+            .register_member(member)
+            .await
+            .map_err(ManagerError::IdentityManager)
+    }
+
+    pub async fn key_bundle_expired(&self) -> bool {
```

Comment:
Here we should mention in docs more what this is used for and what the expected flow is